docs/downloadable-formats.rst

```diff
@@ -0,0 +1,40 @@
+Downloadable Formats
```

Comment:
Would "Downloadable Documentation" be a better name?

Reply:
Perhaps, it has nice alliteration